tests/behat/ui.feature

```diff
@@ -8,25 +8,23 @@ Feature: Codechecker UI works as expected
     Given I log in as "admin"
     And I navigate to "Development > Code checker" in site administration
     And I set the field "Path(s) to check" to "<path>"
-    And I set the field "Exclude" to "*/tests/fixtures/*"
```

Comment:
The next test is testing exclusions explicitly, so safe to remove it here.

Reply:
Aha, I think you're right.

Initially, the tests and fixtures were under main tests directory, and I added that exclude to avoid having to modify all the "Files found" results each time new fixtures were added.

But long ago, the tests were moved to moodle/tests (because they are, mostly, phpcs-standard tests), hence the main test directory is now quite stable normally (until I start testing locallib, but that's another story).

So yes, great you changed that, and thanks for the explanation.